text/70_performance_testing.md

```diff
@@ -182,7 +182,7 @@ Determining the values for the lowest threshold will be heavily dependent on the
 
 Remember to track the level of events and the equivalent throughput, and if possible, relevant utilization measures (see next section for more on measuring utilization).  This will allow you to see if there are any patterns in the throughput levels, thus allowing you to extrapolate further than the data points you have collected.  For example, assume that you see that response time is always approximately 500 millisecond up to 100 events per second, but at 150 events per second, response time slows to 800 milliseconds and at 200 events per second, it is measured to be 2500 milliseconds.  We can see some superlinear growth starting at approximately the 100 events per second mark, and developers will then have more information to start tracking down bottlenecks.  Is that the point where data starts being swapped to disk?  Do we have 100 threads, and this is the point where the thread pool runs dry?  Although you may not have these answers up front, having an idea of the growth, and especially a general area where throughput starts to slow down, will make improving the performance much easier.  Just as we discussed regarding filing defects, being more specific is almost always better than being less specific!
 
-Throughput levels are very sensitive to what kind of hardware you are the software on, so even more than with other tests, you will need to ensure that you are running on the same kind of hardware from run to run to ensure valid results.  You may also want to determine throughput using the same software on different hardware configurations, which may also help to track down the cause of any slowdowns.  For example, a system which has extremely reduced throughput when run on a system with slightly less RAM than another may indicate a memory bottleneck.
+Throughput levels are very sensitive to what kind of hardware you are running the software on, so even more than with other tests, you will need to ensure that you are running on the same kind of hardware from run to run to ensure valid results.  You may also want to determine throughput using the same software on different hardware configurations, which may also help to track down the cause of any slowdowns.  For example, a system which has extremely reduced throughput when run on a system with slightly less RAM than another may indicate a memory bottleneck.
```

Comment:
"Throughput levels are very sensitive to what kind of hardware you are the software on," to "Throughput levels are very sensitive to what kind of hardware you are *running* the software on"